### ISSUE TITLE
recall not improving

### ISSUE THREAD

**Issue by eyebies** [WenmuZhou/DBNet.pytorch#17]
I am training the model on custom dataset and recall is still 0 after 28 epochs. Am I missing anything!
![Screenshot from 2020-02-22 20-44-23](https://user-images.githubusercontent.com/3131247/75097593-2ec00880-55b5-11ea-9653-26a253ac78c4.png)

**Comment by WenmuZhou** [comment]
make sure your text of label in not in ignore tages

**Comment by eyebies** [comment]
Thanks!

**Comment by ghost** [comment]
@WenmuZhou @eyebies What do you mean `ignore tages`?

**Comment by eyebies** [comment]
"###" hashes in annotation!
https://github.com/WenmuZhou/DBNet.pytorch/blob/72c1ed45e2a8ec0e4f02cd5e29cb9f7630cd8035/config/icdar2015.yaml#L38
